modules/hub/pages/manual-mode.adoc

```diff
@@ -0,0 +1,56 @@
+[[manual-mode-example]]
+= Manual mode
+
+
+In this example, manual mode is used, where user needs to provide credentials for each {productname} server user wants to connect to.
```

Comment:
I would reword it as something like:

In the `Manual mode`  the user needs to provide credentials for each {productname} server the user wants to connect to.